package.json

```diff
@@ -137,10 +113,12 @@
         "test:int": "tape 'test/integration/**/*.test.js'",
         "test:integration": "sh ./test/integration-runner.sh ./test/integration-runner.env",
         "test:int:narrow": "jest --config './jest.integration.config.js' --coverage --runInBand --testMatch '**/test/int/**/*.(test|spec).js'",
-        "audit:resolve": "SHELL=sh resolve-audit --production",
-        "audit:check": "SHELL=sh check-audit --production",
+        "audit:resolve": "resolve-audit --production",
+        "audit:check": "check-audit --production",
         "dep:check": "npx ncu -e 2",
-        "dep:update": "npx ncu -u"
+        "dep:update": "npx ncu -u",
+        "release": "standard-version --releaseCommitMessageFormat 'chore(release): {{currentTag}} [skip ci]'",
+        "snapshot": "standard-version --no-verify --skip.changelog --prerelease snapshot --releaseCommitMessageFormat 'chore(snapshot): {{currentTag}}'"
```

Comment:
We are missing the `standard-version` config from here: https://github.com/mojaloop/central-ledger/blob/master/package.json#L150

Don't forget to add the `replace` dev dependency.

Reply:
done.